packages/desktop-client/src/components/payees/index.js

```diff
@@ -220,22 +222,20 @@ const PayeeTable = forwardRef(
 );
 
 function PayeeTableHeader() {
-  let borderColor = colors.border;
   let dispatchSelected = useSelectedDispatch();
   let selectedItems = useSelectedItems();
 
   return (
     <View>
       <TableHeader
+        borderColor={theme.tableborder}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏`TableHeader` does not have a `borderColor` prop